README.md

```diff
@@ -20,16 +21,33 @@ The plugin enables itself when Sonic Pi is running and the Ruby filetype is init
 
 * Extension of Ruby syntax to include Sonic Pi terms
 
+
 ### Installation
 
-Prerequisites: [Sonic Pi 2.4+](http://www.sonic-pi.net/), [sonic-pi-cli](https://github.com/Widdershin/sonic-pi-cli/)
+Prerequisites: [Sonic Pi 2.4+](http://www.sonic-pi.net/), and [sonic-pi-cli](https://github.com/Widdershin/sonic-pi-cli/) or similar.
 
 If you use [pathogen](https://github.com/tpope/vim-pathogen) (and you should), simply clone this repo into `~/.vim/bundle/` like so:
 
 `git clone https://github.com/dermusikman/sonicpi.vim.git`
 
 Whenever Sonic Pi is running, and you haven't disabled the `g:sonicpi_enabled` flag in your configs, the plugin will activate. Otherwise, it's a normal Ruby session!
 
+
+### Configuration
+
+`g:sonicpi_keymaps_enabled` can be used to disable the default keybindings.
+
+`g:sonicpi_command` can be used to configure what tool is used to send the
+code to Sonic Pi. The default is `sonic_pi`.
+
+
+## Sonic Pi interfacing tools
+
+* [sonic-pi-cli](https://github.com/Widdershin/sonic-pi-cli/). The default.
+
+* [sonic-pi-pipe](https://github.com/lpil/sonic-pi-tools). Written in Go, so significantly faster and more responsive than sonic-pi-cli.
```

Comment:
Thank you for the PR! I think it is a valuable improvement to the docs.

I'd like this line to read a bit less like an advertisement. Maybe, "Written in Go, and more responsive than sonic-pi-cli."?